core/src/test/java/org/testcontainers/utility/DockerImageNameTest.java

```diff
@@ -0,0 +1,18 @@
+package org.testcontainers.utility;
+
+import org.junit.Test;
+
+public class DockerImageNameTest {
+
+    @Test
+    public void validNames() {
+        DockerImageName.validate("myname:latest"); // no repo
+        DockerImageName.validate("repo/my-name:1.0"); // no repo
+        DockerImageName.validate("repo.foo.com:1234/my-name:1.0"); // no repo
+    }
```

Comment:
Do you know if it's possible to use namespaces with a custom registry? e.g. `repo.foo.com:1234/mynamespace/my-name:1.0`

I can't say I've seen it in the wild, and a quick look around docs and issues has been inconclusive. However, if such a format is possible then it'd be worth having an extra test case.